src/librustc/hir/mod.rs

```diff
@@ -1450,7 +1450,7 @@ pub struct ItemId {
 /// The name might be a dummy name in case of anonymous items
 #[derive(Clone, PartialEq, Eq, RustcEncodable, RustcDecodable, Hash, Debug)]
 pub struct Item {
-    pub name: Name,
+    pub name: Spanned<Name>,
```

Comment:
I think you misunderstood what I said. HIR items are NOT supposed to contain spans.